terraform/aws.tf

```
@@ -0,0 +1,20 @@
+# Thanks @marcosdiez for the suggestion
+# This makes it super-clear which AWS account, arn, and user_id are in use
+# in a way that can be conveniently tracked in the output of CI tools
+provider "aws" {
+  region = "${var.aws_region}"
+}
+
+data "aws_caller_identity" "current" {}
+
+output "account_id" {
+  value = "${data.aws_caller_identity.current.account_id}"
+}
+
```

I'd also add the account alias as output, see: https://github.com/ozbillwang/terraform-best-practices/blob/master/README.md#avoid-hard-coding-the-resources

```suggestion
data "aws_iam_account_alias" "current" {}

output "account_alias" {
  value = "${data.aws_iam_account_alias.current.account_alias}"
}
```
It is useful for when you have some conditions based on the account alias.